test/asyncio_tests/test_examples.py

```diff
@@ -1503,3 +1503,43 @@ def MongoClient(**kwargs):
         assert isinstance(res["encrypted_unindexed"], Binary)
 
         await client_encryption.close()
+
+
+class MotorAWSLambdaExamples(AsyncIOTestCase):
+    @asyncio_test
+    async def test_shared_client(self):
+        # Start AWS Lambda Example 1
+        import os
+
+        from motor import MotorClient
```

Comment:
This should be `from motor.motor_asyncio import AsyncIOMotorClient` since we're targeting the `asyncio` client.